core/Tracker/VisitExcluded.php

```diff
@@ -180,13 +180,26 @@ private function isIpInRange()
         if ($cache->contains($key)) {
             $isInRanges = $cache->fetch($key);
         } else {
-            $isInRanges = $ip->isInRanges($this->getBotIpRanges());
+            if ($this->isChromeDataSaverUsed($ip)) {
+                $isInRanges = false;
+            } else {
+                $isInRanges = $ip->isInRanges($this->getBotIpRanges());
+            }
+
             $cache->save($key, $isInRanges);
         }
 
         return $isInRanges;
     }
 
+    private function isChromeDataSaverUsed(IP $ip)
+    {
+        // see https://github.com/piwik/piwik/issues/7733
+        return !empty($_SERVER['HTTP_VIA'])
+            && false !== strpos(strtolower($_SERVER['HTTP_VIA']), 'chrome-compression-proxy')
+            && $ip->isInRange('66.249.80.0/20');
```

Comment:
here we look at a specific IP range, instead let's apply this logic for all Google bot IP addresses? (reason: Chrome proxy IPs will likely change in the future, and maybe they already use more than this one IP range)